src/internet/intergalacticfm/intergalacticfmservice.cpp

```diff
@@ -129,8 +129,8 @@ void IntergalacticFMServiceBase::RefreshStreamsFinished(QNetworkReply* reply,
   reply->deleteLater();
 
   if (reply->error() != QNetworkReply::NoError) {
-    // TODO(David Sansome): Error handling
-    qLog(Error) << reply->errorString();
+    app_->AddError(QString("Failed to get channel list:\n") +
```

Comment:
This should probably be internationalised properly, e.g., `tr("Failed to get channel list: \n%1").arg(reply->errorString())`

Reply:
Done